ocean_provider/myapp.py

```diff
@@ -2,14 +2,42 @@
 #  SPDX-License-Identifier: Apache-2.0
 
 import os
+from os.path import abspath, dirname
 
 from flask import Flask
 from flask_cors import CORS
+from flask_sieve import Sieve
+from flask_sqlalchemy import SQLAlchemy
+
+from ocean_provider.utils.basics import get_config
 
 app = Flask(__name__)
 CORS(app)
+Sieve(app)
 
 if 'CONFIG_FILE' in os.environ and os.environ['CONFIG_FILE']:
     app.config['CONFIG_FILE'] = os.environ['CONFIG_FILE']
 else:
     app.config['CONFIG_FILE'] = 'config.ini'
+
+
+PROJECT_ROOT = dirname(dirname(abspath(__file__)))
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:////' + os.path.join(
+    PROJECT_ROOT,
+    get_config().storage_path
+)
+
+db = SQLAlchemy(app)
+
+
+class UserNonce(db.Model):
```

Comment:
why is this in `myapp.py`

Reply:
it needs too be before db.create_all(), so that it is recognized. I'll try to find other solutions.

Reply:
@ssallam I moved them to a models file.